src/presenter/films-presenter.js

```diff
@@ -138,14 +146,13 @@ export default class FilmsPresenter {
     render(this.#sortComponent, this.#filmComponent.element, RenderPosition.BEFOREBEGIN);
   }
 
-  #renderFilm(film, filmContainer) {
+  async #renderFilm(film, filmContainer) {
```

Comment:
```suggestion
  #renderFilm(film, filmContainer) {
```
В нём же нет ничего асинхронного.